Navigation app/Navigation app/SecondScreen/SecondGreenViewController.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  SecondGreenViewController.swift
+//  Navigation app
+//
+//  Created by Dmitriy Dobrinin on 11.08.2022.
+//
+
+import UIKit
+
+class SecondGreenViewController: UIViewController {
+
+    @IBAction private func showDetailsButtonDidTab() {
+
+        let storybord = UIStoryboard(name: "Main", bundle: nil)
+        guard let controller = storybord.instantiateViewController(withIdentifier: "SecondVioletViewController") as? SecondVioletViewController else { return }
+
+        let navigation = UINavigationController(rootViewController: controller)
```

Comment:
As I remember, you have navigation controller as tab item in tabbar controller, so you don't needing more navigation here